src/test/java/uk/gov/companieshouse/overseasentitiesapi/mocks/Mocks.java

```diff
@@ -331,6 +331,7 @@ private static List<TrustDataDto> buildTrustDataInFiling(int numberOfTrusts) {
         for (int i = 1; i <= numberOfTrusts; i++) {
             TrustDataDto trustDataDto = new TrustDataDto();
             trustDataDto.setTrustName("Trust Name " + i);
+            trustDataDto.setChReference("ch_reference" + i);
```

Comment:
These mocks must be used by a unit test somewhere. Need to find which one(s) and update the assertions to check that the 'ch_reference' field is set correctly (I think).

Reply:
I will add it

Reply:
in reality, it is not necessary
I will remove it